src/main/java/org/apache/maven/report/projectinfo/PluginManagementReport.java

```diff
@@ -255,5 +283,41 @@ public int compare( Plugin a1, Plugin a2 )
                 }
             };
         }
+        
+        private boolean isExcluded( Artifact pluginArtifact )
+        {
+            if ( excludes == null )
+            {
+                return false;
+            }
+
+            for ( String pattern : excludes )
+            {
+                String[] subStrings = pattern.split( ":" );
+                subStrings = StringUtils.stripAll( subStrings );
+                String resultPattern = StringUtils.join( subStrings, ":" );
+
+                if ( compareDependency( resultPattern, pluginArtifact ) )
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        /**
+         * Compares the given pattern against the given artifact. The pattern should follow the format
+         * <code>groupId:artifactId:type:classifier:version</code>.
+         * 
+         * @param pattern The pattern to compare the artifact with.
+         * @param artifact the artifact
+         * @return <code>true</code> if the artifact matches the pattern
+         */
+        protected boolean compareDependency( String pattern, Artifact artifact )
+        {
```

Comment:
Use org.apache.maven.shared.artifact.filter.PatternExcludesArtifactFilter instead

Reply:
I didn't knew that class...very handy!